testng/src/test/java/io/cucumber/testng/TestNGCucumberRunnerTest.java

```diff
@@ -73,6 +76,25 @@ public void provideScenariosIsIdempotent() {
                 .filter(TestRunFinished.class::isAssignableFrom).count());
     }
 
+    @Test
+    public void runWithCustomeOptions() {
+
+        Map<String, String> customOptions = new HashMap<String, String>();
+        customOptions.put(Constants.PLUGIN_PROPERTY_NAME, "io.cucumber.testng.TestNGCucumberRunnerTest$Plugin");
+        testNGCucumberRunner = new TestNGCucumberRunner(RunWithCustomOptions.class);
+
+        testNGCucumberRunner.provideScenarios();
+        testNGCucumberRunner.provideScenarios();
+        testNGCucumberRunner.finish();
+
+        assertEquals(1, events.stream()
+                .map(Object::getClass)
+                .filter(TestRunStarted.class::isAssignableFrom).count());
+        assertEquals(1, events.stream()
```

Comment:
This test only passes because the `Plugin.events` are not reset between tests.

Reply:
yes, the test is wrong, I'm updating